aws-s3-transfer-manager/src/operation/download/service.rs

```diff
@@ -64,7 +75,7 @@ async fn download_chunk_handler(
 
     let bytes = body
         .collect()
-        .instrument(tracing::debug_span!("collect-body", seq = seq))
+        .instrument(tracing::debug_span!("download-chunk-collect-body", seq))
```

Comment:
We are not instrumenting the send-download-chunk few lines above?

Reply:
I thought it was more useful to wrap this function in a parent span, so send + collect are grouped together

The HTTP requests where send() doesn't do all the work are weird ... spanwise